src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodcount/Example1.java

```diff
@@ -0,0 +1,31 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodCount"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodcount;
+
+// xdoc section -- start
+class Example1 {
+
+  public void outerMethod1(int i){}
+  public void outerMethod2(){}
+  public void outerMethod3(String str){}
+
+  private void outerMethod4(){
+    Runnable r = (new Runnable() {
+      public void run() {} // NOT counted towards ExampleClass
+    });
+  }
+
+  private void outerMethod5(int i){}
+  void outerMethod6(int i , int j){}
+
+  public static class InnerExample{
+    public void InnerMethod1(){} // NOT counted towards ExampleClass,
```

Comment:
`innerMethod1()`

Same for next method

Reply:
Done